docs/getting-started-with-scalardb.md

```diff
@@ -36,206 +185,59 @@ You can create a JSON file `emoney.json` with the JSON below.
-    }
-  }
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-  public void pay(String fromId, String toId, int amount) throws TransactionException {
-    // Start a transaction
-    DistributedTransaction tx = manager.start();
-
-    try {
-      // Retrieve the current balances for ids
-      Get fromGet =
-              Get.newBuilder()
-                      .namespace(NAMESPACE)
-                      .table(TABLENAME)
-                      .partitionKey(Key.ofText(ID, fromId))
-                      .build();
-      Get toGet =
-              Get.newBuilder()
-                      .namespace(NAMESPACE)
-                      .table(TABLENAME)
-                      .partitionKey(Key.ofText(ID, toId))
-                      .build();
-      Optional<Result> fromResult = tx.get(fromGet);
-      Optional<Result> toResult = tx.get(toGet);
-
-      // Calculate the balances (it assumes that both accounts exist)
-      int newFromBalance = fromResult.get().getInt(BALANCE) - amount;
-      int newToBalance = toResult.get().getInt(BALANCE) + amount;
-      if (newFromBalance < 0) {
-        throw new RuntimeException(fromId + " doesn't have enough balance.");
-      }
-
-      // Update the balances
-      Put fromPut =
-              Put.newBuilder()
-                      .namespace(NAMESPACE)
-                      .table(TABLENAME)
-                      .partitionKey(Key.ofText(ID, fromId))
-                      .intValue(BALANCE, newFromBalance)
-                      .build();
-      Put toPut =
-              Put.newBuilder()
-                      .namespace(NAMESPACE)
-                      .table(TABLENAME)
-                      .partitionKey(Key.ofText(ID, toId))
-                      .intValue(BALANCE, newToBalance)
-                      .build();
-      tx.put(fromPut);
-      tx.put(toPut);
-
-      // Commit the transaction (records are automatically recovered in case of failure)
-      tx.commit();
-    } catch (Exception e) {
-      tx.abort();
-      throw e;
-    }
-  }
-
-  public int getBalance(String id) throws TransactionException {
-    // Start a transaction
-    DistributedTransaction tx = manager.start();
-
-    try {
-      // Retrieve the current balances for id
-      Get get =
-              Get.newBuilder()
-                      .namespace(NAMESPACE)
-                      .table(TABLENAME)
-                      .partitionKey(Key.ofText(ID, id))
-                      .build();
-      Optional<Result> result = tx.get(get);
-
-      int balance = -1;
-      if (result.isPresent()) {
-        balance = result.get().getInt(BALANCE);
-      }
-
-      // Commit the transaction
-      tx.commit();
-
-      return balance;
-    } catch (Exception e) {
-      tx.abort();
-      throw e;
-    }
-  }
-
-  public void close() {
-    manager.close();
-  }
-}
-```
-
-Now we can run the application.
-
-- Charge `1000` to `user1`:
+- Credit **1000** to **customer1** by running the following command:
```

Comment:
I changed `user1` to `customer1` because a customer-merchant relationship is common in commerce.